tests/code/test_users.py

```diff
@@ -322,3 +321,27 @@ def test_getUserBGCheckHistory():
 
 
         transaction.rollback()
+
+@pytest.mark.integration
+def test_getBannedUsers():
+    with mainDB.atomic() as transaction:
+        with app.app_context():
+
+            # Create test user
+            testusr = User.create(username = 'usrtst',
+                                  firstName = 'Test',
+                                  lastName = 'User',
+                                  bnumber = '03522492',
+                                  email = 'usert@berea.deu',
+                                  isStudent = True)
+            # Ban user from program 1
+            banUser(1, User.get_by_id("usrtst"), "nope", "2022-11-29", "ramsayb2")
```

Comment:
Make the date further in the future as this will break in 2 months.

Reply:
Changed to 2050 which is a while from now